visual-recognition/src/main/java/com/ibm/watson/visual_recognition/v4/model/UpdateObjectMetadata.java

```diff
@@ -37,11 +37,9 @@ public Builder() {}
      * Instantiates a new builder with required properties.
      *
      * @param object the object
-     * @param count the count
      */
-    public Builder(String object, Long count) {
+    public Builder(String object) {
       this.object = object;
-      this.count = count;
```

Comment:
is this good?

Reply:
fixed in [0f6d07af0fa398a550ed2df90eff4f06f1f46c93](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/0f6d07af0fa398a550ed2df90eff4f06f1f46c93)